### PR TITLE
fix `SCT_TAGRET_UPGRADE_VERSION` to be set automatically 

### PR DESCRIPTION
**Pull request by fruch**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [x] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [x] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (`hydra unit-tests`)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
@juliayakovlev FYI, a change that we can inspect the repo/list and remove the need to use the branch name for this

File: docker/env/version [hunk 1]
```diff
@@ -1 +1 @@
-0.39
+0.39-a
```
Comment (bentsi) [inline docker/env/version]: remove the `-a`

File: sdcm/version_utils.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import requests
```
Comment (bentsi) [inline sdcm/version_utils.py]: the dir structure should be as follows:
sdcm/utils/scylla_version.py
sdcm/utils/common.py (move and rename `utils.py`)

File: docker/env/version (hunk 1 above)
Comment (fruch) [reply docker/env/version]: change to 0.40

File: sdcm/version_utils.py (hunk 2 above)
Comment (fruch) [reply sdcm/version_utils.py]: changed

**Pull request merged**